WindowsGSM/GameServer/Action/Install.cs

```diff
@@ -223,6 +223,7 @@ public async Task<bool> IsSuccess(Process process, string serverGame, string ser
 
         public async void CreateServerConfigs(string serverGame, string serverName)
         {
+            await Task.Yield();
```

Comment:
This will cause a UI bug. The server can not display on the server table after the server had installed.

Reply:
If that is the case, then the warning should be disabled with a pragma